.gitlab/includes/common_spack_pipeline.yml

```
@@ -8,7 +8,7 @@ include:
   - local: '.gitlab/includes/common_pipeline.yml'
 
 variables:
-  SPACK_COMMIT: develop-2024-08-18
+  SPACK_COMMIT: b9e4e98f15b4a07814adf8b22da19e631f2749e4
```

Updating to the latest tag with a date, so it's easier to see how old the tag is.

```suggestion
  SPACK_COMMIT: develop-2024-09-01
```